modules/api/src/main/java/io/fluo/api/client/FluoAdmin.java

```diff
@@ -24,7 +24,7 @@
   /**
    * Specifies Fluo initialization options such as clearing Zookeeper or existing Accumulo table.
    */
-  public static class InitOpts {
+  class InitOpts {
```

Comment:
interesting, I suppose an inner class in an interface can not be anything but static